Makefile

```diff
@@ -97,6 +97,11 @@ vtest: .develop
 vvtest: .develop
 	@pytest -vv
 
+.PHONY: cov-dev
+cov-dev: .develop
+	@pytest --cov-report=html
+	@echo "open file://`pwd`/htmlcov/index.html"
```

Comment:
This is macOS-only. There should be a fallback to `xdg-open` for GNU/Linux if we want this to be useful in general.

Reply:
fixed